src/main/scala/de/heikoseeberger/sbtheader/HeaderPlugin.scala

```diff
@@ -54,6 +54,7 @@ object CommentStyleMapping {
 
 object HeaderKey {
   val headers = settingKey[Map[String, (Regex, String)]]("Header pattern and text by extension; empty by default")
+  val excludes = settingKey[Seq[String]]("File patterns for files to be excluded; empty by default")
```

Comment:
AFAIU `Predef` only imports the immutable variants of `Set` and `Map`. If you write `val s = Seq("Hello")` you will get a `scala.collection.Seq`. Since we have our own import of `scala.collection.immutable.Seq` in our package object, we get a `de.heikoseeberger.sbtheader.Seq` which really is a `scala.collection.immutable.Seq`.

Have a look at sbt-test/sbt-header/excludes/test.sbt. If I change line 7
from `excludes := scala.collection.immutable.Seq(`
to `excludes := Seq(`

This causes the build to fail with:

```
[error] /private/var/folders/66/6_p15z9j15b6gmgyc19g42cw0000gn/T/sbt_27f5cba3/excludes/test.sbt:7: error: type mismatch;
[error]  found   : Seq[String]
[error]  required: de.heikoseeberger.sbtheader.Seq[String]
[error]     (which expands to)  scala.collection.immutable.Seq[String]
[error] excludes := Seq(
[error]                ^
[info] [error] Type error in expression
```

I think this will confuse users, so we should probably use a different collection type for the settings key. You proposed `List` which would work, but I don't see a reason to allow duplicates. So how about `Set`?